src/lib.rs

```diff
@@ -474,16 +65,14 @@ macro_rules! row_col_vec {
     };
 }
 
-#[derive(Parser)]
-#[grammar = "coordinate.pest"]
-struct CoordinateParser;
-
 // macro for easily defining a coordinate
 // either absolutely or relative to it's parent coordinate
 // TODO: this code is messy, can be optimized more later 
+#[macro_export]
 macro_rules! coord {
```

Comment:
can we move the macros related to `Coordinate` into coordinate.rs?

Reply:
possibly... I had them in utils.rs originally and couldn't get it to work but it should technically be possible.